openshift-playbook.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Configure OpenShift deployment for linux-system-roles CI testing
+  hosts: localhost
```

Comment:
This should probably be a variable, too.

Reply:
What shall I name the variable?  This host is the host where I want the playbook to be run, so that it has access to the `oc` command and your current OpenShift authentication context.

Reply:
That is, this host is not the host where I want to install something into.  I suppose this could be something other than `localhost` if you wanted to run ansible on your local workstation, that uploaded the files/scripts to some sort of jump host, which has the `oc` commands and your OpenShift authentication context?  Seems unlikely - I would rather leave this as `localhost` unless there is some use case I am unaware of.

Reply:
Ah, I see. Then it does not need to be a variable but the README should mention how to select the cluster for the playbook.